packages/react-native-nitro-image/ios/NitroExampleView.mm

```diff
@@ -0,0 +1,90 @@
+#import <React/RCTComponentViewFactory.h>
+#import <React/UIView+ComponentViewProtocol.h>
+#import <UIKit/UIKit.h>
+#import <react/renderer/componentregistry/ComponentDescriptorRegistry.h>
+
+#import <React/RCTViewComponentView.h>
+#import <jsi/jsi.h>
+#import <react/renderer/components/view/ConcreteViewShadowNode.h>
+#import <react/renderer/components/view/ViewEventEmitter.h>
+#import <react/renderer/components/view/ViewProps.h>
+#import <react/renderer/core/ConcreteComponentDescriptor.h>
+#import <react/renderer/core/PropsParserContext.h>
+
+#import "HybridTestObjectCppSpec.hpp"
+//#import "JSIConverter.hpp"
+
+// for prop conversion
+#import <react/renderer/core/propsConversions.h>
+
+
+using namespace facebook;
+using namespace facebook::react;
+using namespace margelo::nitro;
+
+// We have to inherit from ViewProps so that we can
+class CustomViewProps final : public ViewProps {
+public:
+  CustomViewProps() = default;
+  CustomViewProps(const PropsParserContext& context, const CustomViewProps& sourceProps, const RawProps& rawProps)
+      : ViewProps(context, sourceProps, rawProps) {
+        if (rawProps.isEmpty()) {
+          // There is some code path in RawPropsParser.cpp where we expect to collect all possible keys (prop names) upfront
+          // during the init phase. Calling this .at() method while the rawProps is empty, will cause this prop value to be indexed …
+          // Otherwise we'd get a crash when trying to access it later.
+          // We should really also try to merge the PR i have for providing alternative RawProps parser !!
+          rawProps.at("nativeProp", nullptr, nullptr);
```

Comment:
Not sure if I fully understand - is this still needed? Do I need to do this for _every_ prop?

Reply:
but yeah, it has to be called at this point once, as the `.at()` method will insert the props key into a map. If later on the prop name is not in this key map it will always return null. Super weird I know.
my main idea is that we are able to write our own prop parser to work around this